config/samples/kustomization.yaml

```
@@ -6,4 +6,4 @@ resources:
 - datasciencecluster_v1_datasciencecluster.yaml
 - dscinitialization_v1_dscinitialization.yaml
 - services_v1alpha1_auth.yaml
-- hardwareprofile_v1alpha1_hardwareprofile.yaml
\ No newline at end of file
+- hardwareprofile_v1_hardwareprofile.yaml
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)